src/views/MakingARequest/index.tsx

```diff
@@ -26,7 +26,9 @@ const MakingARequest = () => {
           <li>IT Governance</li>
         </BreadcrumbNav>
 
-        <PageHeading>{t('heading')}</PageHeading>
+        <PageHeading className="margin-bottom-4 margin-top-5">
```

Comment:
I am hesitant about this spacing unless we really need to. A little while back, we tried to standardize the amount of spacing between the `h1` and surrounding elements. The `PageHeading` component has the appropriate spacing that we have on all pages and this doesn't look like a unique use case. It seems like we should stay consistent with what we have throughout the application.

Reply:
Ok, sounds like Monica is ok with leaving this spacing out, so I'll remove it.  It would be good to get the Figma designs and our global styles aligned.